source/simulator/helper_functions.cc

```diff
@@ -1615,17 +1617,7 @@ namespace aspect
     LinearAlgebra::BlockVector distributed_reaction_vector (introspection.index_sets.system_partitioning,
                                                             mpi_communicator);
 
-    // we use a different (potentially smaller) time step than in the advection scheme,
-    // and we want all of our reaction time steps (within one advection step) to have the same size
-    const unsigned int number_of_reaction_steps = std::max(static_cast<unsigned int>(time_step / parameters.reaction_time_step),
-                                                           std::max(parameters.reaction_steps_per_advection_step,1U));
-
-    const double reaction_time_step_size = time_step / static_cast<double>(number_of_reaction_steps);
-
-    Assert (reaction_time_step_size > 0,
-            ExcMessage("Reaction time step must be greater than 0."));
-
-    pcout << "   Solving composition reactions... " << std::flush;
+    pcout << "   Solving composition reactions with SUNDIALS... " << std::flush;
```

Comment:
I'd just drop this change -- this will minimize the difference in the output files.